plugins/permission-node/src/ServerPermissionClient.ts

```diff
@@ -74,23 +78,34 @@ export class ServerPermissionClient implements PermissionsService {
       auth,
       permissionClient,
       permissionEnabled,
+      pluginId: options.pluginId,
     });
   }
 
   private constructor(options: {
     auth: AuthService;
     permissionClient: PermissionClient;
     permissionEnabled: boolean;
+    pluginId?: string;
   }) {
     this.#auth = options.auth;
     this.#permissionClient = options.permissionClient;
     this.#permissionEnabled = options.permissionEnabled;
+    this.#pluginId = options.pluginId;
```

Comment:
This isn't needed anymore right?

Reply:
Correct! I thought I addressed that. Fixed.